packages/web-components/src/card/card.template.ts

```diff
@@ -0,0 +1,30 @@
+import { ElementViewTemplate, html, when } from '@microsoft/fast-element';
+import type { Card } from './card.js';
+
+/**
+ * The template for the Fluent Card web-component.
+ * @public
+ */
+export function cardTemplate<T extends Card>(): ElementViewTemplate<T> {
+  return html<T>`
+    <template
+      role="group"
+      ?interactive="${x => x.interactive}"
+      ?selected="${x => x.selected}"
```

Comment:
Same question here regarding why the hook is needed if we intend to use `aria-selected`

Reply:
It's not.. misunderstanding on my part.